chapters/functions.tex

```diff
@@ -2243,13 +2243,15 @@ \subsection{Annotations for External Libraries and Include Files}\label{annotati
     input Real x;
     output Real y;
     external "C"
-    y=ExternalFunc1_ext(x) annotation(Library="ExternalLib1",Include="#include \"ExternalFunc1.h\"");
+    y=ExternalFunc1_ext(x) annotation(Library="ExternalLib1",
+      Include="#include \"ExternalFunc1.h\"");
```

Comment:
How about breaking the line before the annoation?
```
  output Real y;
external "C"
  y = ExternalFunc1_ext(x)
    annotation(…);
```

Reply:
I don't think it is sufficient. 
We could still do it, and also break before Library.
However, then it would be a different style and should be done more consistently.

Reply:
Similar to the one below, an alternative here could be this:
```
  output Real y;
external "C"
  y = ExternalFunc1_ext(x)
    annotation(Library="ExternalLib1",
               Include="#include \"ExternalFunc1.h\"");
```